config/inputs/io_nvme_input.txt

```diff
@@ -1,26 +1,25 @@
 [io_nvme]
-device = "nvme0"
```

Comment:
we need device parameter to cover the 8 nvme functinal test in cfg

Reply:
Added the changes specific to nvme file